src/main/java/duke/commands/CreateTodoCommand.java

```diff
@@ -0,0 +1,26 @@
+package duke.commands;
+
+import duke.data.TaskList;
+import duke.data.tasks.ToDos;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Creates and adds a ToDoTask
+ */
+public class CreateTodoCommand extends Command {
+    private final String name;
+
+    /**
```

Comment:
should you also give a brief summary of what the method does first in javadocs first line?